api/all/README.md

```diff
@@ -16,3 +16,8 @@
 From the root of the repo run `./gradlew clean :api:jmh` to run all the benchmarks
 or run `./gradlew clean :api:jmh -PjmhIncludeSingleClass=<ClassNameHere>`
 to run a specific benchmark class.
+
+---
+#### Logging API mis-use
+Define `logging.level.ApiUsageLogging=TRACE` to enable TRACE logging for possible invalid or
```

Comment:
Does this need to change now that the logger is named `io.opentelemetry.ApiUsageLogging` instead of `ApiUsageLogging`?

Reply:
Yes, sorry, I missed that